c/bubble_sort.c

```diff
@@ -0,0 +1,34 @@
+/*
+BUBBLE SORT with Time Complexity : n
+int main()
```

Comment:
Please uncomment the main code for both bubble_sort.c and insertion_sort.c.

Reply:
What we want to do is have code which just need to be compiled without editing any file.